data/kushvahasumit.json

```
@@ -0,0 +1,37 @@
+{
+  "name" : "sumit",
+  "displayStatusPublic" : true,
+  "type" : "personal",
+  "bio" : "Undergraduate || new for open source || enthusiast about tech world || web dev",
+  "avatar" : "https://github.com/kushvahasumit.png",
+  "links" : [
+  {
+  "name" : "follow me on Linkedin",
+  "url" : "https://www.linkedin.com/in/sumit-kumar-4a1344254",
+  "icon" : "FaLinkedin"
+},
+ {
+  "name" : "follow me on Twitter",
+  "url" : "https://twitter.com/ur_samy007",
+  "icon" : "FaTwitter"
+},
+{
+  "name" : "follow me on GitHub",
+  "url" : "https://github.com/kushvahasumit",
+  "icon" : "FaGitHub"
+},
+{
+  "name" : "subscribe my YouTube",
+  "url" : "https://youtube.com/@samcode130",
+  "icon" : "FaYoutube"
+},
+{
+  "name" : "my Hashnode blog",
+  "url" : "https://hashnode.com/@kushvaha687",
+  "icon" : "FaHashnode"
```

This is not the right icon for hashnode. Try,
```suggestion
  "icon" : "SiHashnode"
```